app/forms/openid_connect_token_form.rb

```diff
@@ -30,11 +30,17 @@ def initialize(params)
       instance_variable_set(:"@#{key}", params[key])
     end
 
-    @identity = Identity.where(session_uuid: code).first
+    session_expiration = Figaro.env.session_timeout_in_minutes.to_i.minutes.ago
```

Comment:
this isn't for right now, but I feel like given the number of methods required on this it would be cool to have a little utility class that changes figaro values into minutes.